### ISSUE TITLE
Add support for XLOOKUP formula in OpenXML SDK

### ISSUE THREAD

**Issue by tauheedul** [OfficeDev/Open-XML-SDK#1189]
** Description **
Please add support for XLOOKUP formula parsing and calculation in Strict Open XML XLSX file output

**Information**

- .NET Target: All
- DocumentFormat.OpenXml Version: 2.17.1

**Repro**

```csharp
Example formula
=XLOOKUP(A1, A3:A5, B3:B5)
``` 

**Observed**

Lookup is not calculated in XLSX output
![image](https://user-images.githubusercontent.com/9335220/182652778-67aef7a4-963b-44ce-ab7a-d271b92ca1e0.png)

**Expected**

Support for formula syntax
=XLOOKUP(lookup_value, lookup_array, return_array, [if_not_found], [match_mode], [search_mode])

**Comment by ThomasBarnekow** [comment]
@tauheedul, can you be (much) more specific about what you are trying to achieve? For example, do you want the Open XML SDK to compute the result of the `XLOOKUP` function? That would go beyond the scope of this library.

**Comment by tauheedul** [comment]
> @tauheedul, can you be (much) more specific about what you are trying to achieve? For example, do you want the Open XML SDK to compute the result of the `XLOOKUP` function? That would go beyond the scope of this library.

Hello @ThomasBarnekow The OpenXML SDK should be able to read the calculated output of the XLOOKUP function when parsing the document in .NET. I believe it is possible to read the output of a VLOOKUP with Open XML. 
This request is to extend the capability to include XLOOKUP

Thank you.

**Comment by tomjebo** [comment]
@tauheedul, 

The SDK doesn't currently compute any formulas. I think what's happening is that there is a problem with the formula in your first case. I tested this out and if I provide unique values for A3-A5 (versus all being 10001), the sheet opens with the value "TEST1" filled in. Also, I assume you're not providing the <v> element under the <f> element in the C1 cell markup. That's ok but if you want the previously calculated (by Excel) value, you would have to include that <v> element. Like this: 

```
      <c r="C1" s="1" t="str">
        <f>_xlfn.XLOOKUP(A1, A3:A5, B3:B5)</f>
        <v>something</v>
      </c>
```

Basically, whatever you provide there will be displayed until you select the C1 cell and hit enter or otherwise cause Excel to re-evalutate the formula. 

But I believe the problem is that the formula was not able to operate properly given the input table. 

Let me know if this helps.

**Comment by tauheedul** [comment]
Hello @tomjebo thank you. Your comments have been helpful. I will close this request.
